streaming/src/main/scala/org/apache/spark/streaming/api/java/JavaPairDStream.scala

```diff
@@ -848,6 +848,6 @@ object JavaPairDStream {
 
   def scalaToJavaLong[K: ClassTag](dstream: JavaPairDStream[K, Long])
   : JavaPairDStream[K, JLong] = {
-    DStream.toPairDStreamFunctions(dstream.dstream).mapValues(new JLong(_))
+    DStream.toPairDStreamFunctions(dstream.dstream).mapValues(JLong.valueOf)
```

Comment:
Changed for consistency; `valueOf` is slightly more efficient than the constructor since the JDK actually caches `Long` objects for small values.